functorch/__init__.py

```diff
@@ -5,25 +5,24 @@
 # LICENSE file in the root directory of this source tree.
 import torch
 
-from torch._functorch.deprecated import (
-    combine_state_for_ensemble,
+from torch._functorch.apis import grad, grad_and_value, vmap
```

Comment:
We want people to move away from `import functorch`, so these should all still be from `torch._functorch.deprecated` import blah, right?